ydb/library/yql/minikql/comp_nodes/mkql_block_map_join.cpp

```diff
@@ -257,15 +257,15 @@ using TState = TBlockJoinState<RightRequired>;
                 while (blockState.IsNotFull() && blockState.NextRow()) {
                     const auto key = MakeKeysTuple(blockState);
                     if constexpr (WithoutRight) {
-                        if (key && Dict_.Contains(key) == RightRequired) {
+                        if ((key && Dict_.Contains(key)) == RightRequired) {
                             blockState.CopyRow();
                         }
                     } else if constexpr (RightRequired) {
                         if (NUdf::TUnboxedValue lookup; key && (lookup = Dict_.Lookup(key))) {
                             blockState.MakeRow(lookup);
                         }
                     } else {
-                        blockState.MakeRow(Dict_.Lookup(key));
+                        blockState.MakeRow(key ? Dict_.Lookup(key) : NUdf::TUnboxedValue {} );
```

Comment:
Typo: I'd rather follow the existing code style (e.g. [here](https://github.com/ydb-platform/ydb/blob/4f53f25259050df45a33a704d7e02daac18a2a5a/ydb/library/yql/minikql/comp_nodes/mkql_block_map_join.cpp#L418)), but I don't insist here, so feel free to ignore.